espnet/nets/pytorch_backend/e2e_asr_transformer.py

```diff
@@ -213,6 +223,14 @@ def forward(self, xs_pad, ilens, ys_pad):
             if not self.training:
                 self.ctc.softmax(hs_pad)
 
+            if self.intermediate_ctc_weight > 0 and self.intermediate_ctc_layers:
+                for hs_intermediate in hs_intermediates:
+                    # assuming hs_intermediates and hs_pad has same length / padding
+                    loss_inter = self.ctc(hs_intermediate.view(batch_size, -1, self.adim), hs_len, ys_pad)
+                    loss_intermediate_ctc += loss_inter
+
+                loss_intermediate_ctc /= len(self.intermediate_ctc_layers)
+
```

Comment:
```suggestion
            if self.intermediate_ctc_weight > 0:
                loss_ctc = (1 - self.intermediate_ctc_weight) * loss_ctc + self.intermediate_ctc_weight * loss_intermediate_ctc
```

Reply:
This would alter the perceived loss curve (in log or tensorboard), as it becomes weighted sum of two losses, not the original CTC loss. Is it ok?
Usually the original CTC loss becomes smaller with intermediate CTC, but the weighted sum of two losses would become larger than before (as intermediate loss >= CTC loss).